### DISCUSSION TITLE
Is building while game is paused OP?  Espacially with Tramiel Upgrade Lvl 6

### DISCUSSION THREAD

**Discussion by klaviartur** [ochadenas/cpudefense#146] [Polls]
I personally think, buying, upgrading and selling Chips while the game is paused is overpowered.

While it was not that obvious to me in lower levels, it became very apparent when I had upgraded Jack Tramiel to Level 6 (max, 110% refund)

Both features together turn the requirement for skill into a requirement for patience:
With enough time on hand you can now repeatedly buy and sell chips in paused mode until you get enough money  to max out all slots with the perfect chips even before the first attacker arrives on scene.

I have multiple ideas how to address this issue, my favourite being the most complex one:
To kill the OP feature and simultaniously keep the opportunity to earn money from trading if you are brave and skilled enough, i would like to see building cost more when paused (e.g. 5 %pt) and accordingly selling should have a lower refund while paused (10 %pt), so that you can get a maximum of 100% refund when paused even with Jack Tramiel maxed out.

**Comment by ochadenas** [comment]
The problem is only Tramiel, if I see this correctly. We had this discussed already: I thought that it would be somewhat funny to allow refund a bit more than the price paid. But it seems to cause more problems than it's worth, and I don't insist on this feature ...

So the easiest way to solve this is to limit Tramiel to 100%, which means your solution # 4.

**Comment by Cwpute** [comment]
Although i'm against your favored solution (things cwst more and sell fwr less when paused), i just love the idea of having the gameplay change, even a little bit, when you're paused. You could expand on it and give a bonus when fast-forwarding etc…
Makes you wonder if there is something interesting de to with that here 🤔

**Comment by The-JMFB** [comment]
I feel like the amount of time required to really exploit the resell strategy is so high that it's not really overpowered. If a player wanted to invest that kind of time and energy into doing that, then I say let them. This is a solo game, not a competitive game. It's not like it gives someone an advantage in a competition to simply have more time available to exploit this than someone else.

Strategy is strategy, whether you approve of the strategy or not. Do I think it's a bit pathetic for someone to invest hours in raising information to max out every chip slot with this strategy? Sure. Do I respect the commitment to the enormous time suck to do it? Absolutely.

If any change was made, I'd vote for limiting the refund to 100%. Pause building is essential in this game, especially in endless mode.

**Comment by ochadenas** [comment]
In 1.34, Tramiel is limited to level 5, corresponding to a 100% refund.
